django_feature_flip/tests.py

```diff
@@ -78,8 +106,68 @@ def test_activate_group_for_non_existent_feature(self):
 
     def test_activate_group_for_non_existent_group(self):
         FeatureFactory(name='my_feature')
-
         try:
             self.feature_flip.activate_group('my_feature', 'non_existent_group')
         except FeatureFlip.FeatureFlipError as error:
             self.assertEqual(error.message, 'Group non_existent_group does not exist')
+
+    def test_enabled_for_enabled_actor(self):
+        feature = FeatureFactory()
```

Comment:
I think you execute this in all of the unit tests, maybe you should add this in a setup function, something like:
```
this.feature = FeatureFactory()
```
And remove this from each unit test